edx/edx-ora2
# [BD-05] [BB-3037] Convert ORA to pluggable app

Convert ORA to a pluggable Django app so that it's constituent apps do not rely on being loaded by edx-platform through envs/common.py

**Dependencies**: https://github.com/edx/edx-platform/pull/25304

**Testing instructions**:

1. Checkout the PR and install this package
2. Thoroughly check and make sure all functionality of ORA still works (File upload, workflows etc)

**Reviewers**
